namelists/echam/6.3.05p2-wiso/NUDG/namelist.echam

```diff
@@ -0,0 +1,131 @@
+&runctl
+    dt_start = 2013, 12, 31, 23, 52, 30
+    dt_stop = 2019, 12, 31, 23, 52, 30
+    putrerun = 1, 'months', 'last', 0
+    lfractional_mask = .false.
+    lresume = .true.
+    out_datapath = './'
+    out_expname = 'NUDGING_ERA5_T63L47'
+    rerun_filetype = 4
+    delta_time = 450
+    putdata = 6, 'hours', 'last', 0
+    lamip = .true.
+    lnudge = .true.
+    lcouple = .false.
+    lcouple_co2 = .false.
+    default_output = .false.
+    l_orbvsop87 = .true.
+/
+&ndgctl
+    dt_nudg_start = 2013, 12, 31, 23, 52, 30
+    dt_nudg_stop  = 2019, 12, 31, 23, 52, 30
+    inudgformat = 2
+    ndg_file_nc = 'ndg%y4%m2.nc'
+    nsstinc = 0
+    lnudgcli = .false.
+    lnudgini = .false.
+/
+&wisoctl
+    lwiso = .false.
+    lwiso_rerun = .false.
+    nwiso = 0
+/
+&parctl
+    nproca = 24
+    nprocb = 24
+    nprocar = 0
+    nprocbr = 0
+/
+&submodelctl
+    lmethox = .true.
+/
+&mvstreamctl ! 'default spm glm g3bm'
+    filetag = 'echam'
+    source = 'sp', 'gl', 'g3b'
+/
+&mvstreamctl ! 'default co2m'
+    filetag = 'co2'
+    source = 'co2'
+/
+&mvstreamctl ! 'default surfm'
+    filetag = 'surf'
+    source = 'surf'
+/
+&mvstreamctl ! 'default tracerm'
+    filetag = 'tracer'
+    source = 'tracer'
+/
+&mvstreamctl ! 'default jsbachm'
+    filetag = 'jsbach'
+    source = 'jsbach'
+/
+&mvstreamctl ! 'default landm'
+    filetag = 'land'
+    source = 'land'
+/
+&mvstreamctl ! 'default vegm'
+    filetag = 'veg'
+    source = 'veg'
+/
+&mvstreamctl ! 'default yassom'
+    filetag = 'yasso'
+    source = 'yasso'
+/
+&set_stream ! 'default jsbach'
+    stream = 'jsbach'
+    lpost = 0
+/
+&set_stream ! 'default land'
+    stream = 'land'
+    lpost = 0
+/
+&set_stream ! 'default veg'
+    stream = 'veg'
+    lpost = 0
+/
+&set_stream ! 'default yasso'
+    stream = 'yasso'
+    lpost = 0
+/
```

Comment:
We might need to check that these output file settings match up with the movement (copy, move, or link) of output to the correct folders